docker/presets/llama-2/Dockerfile

```diff
@@ -26,8 +26,10 @@ RUN pip install 'uvicorn[standard]'
 ARG WEIGHTS_PATH
 ARG MODEL_TYPE
 ARG VERSION
-# Write the version to a file
-RUN echo $VERSION > /workspace/llama/version.txt
+ARG IMAGE_NAME
+
+# Write metadata to model_info.txt file
+RUN echo -e "$VERSION\n$IMAGE_NAME" > /workspace/llama/model_info.txt
```

Comment:
small nit - can the format of this be
$IMAGE_NAME:$VERSION

Reply:
It is fixed